client/web/src/search/home/QueryExamplesHomepage.module.scss

```
@@ -93,6 +93,10 @@
     }
 }
 
+.tab-panel {
+    min-height: 335px;
```

Let's use `rem` so that if the user changes their default font size we scale with that. Also add a comment explaining why this is needed.

```suggestion
    // Set a minimum height so the page doesn't shift when changing tabs
    min-height: 21rem; 
```

---

Good call! Thanks Juliana